src/sentry/models/issueviews.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from django.db.models import UniqueConstraint
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import region_silo_model
+from sentry.db.models.base import DefaultFieldsModel
+from sentry.db.models.fields.foreignkey import FlexibleForeignKey
+from sentry.models.savedsearch import SortOptions
+
+
+@region_silo_model
+class IssueViews(DefaultFieldsModel):
+    """
+    A model for a user's view of the issue stream
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+    name = models.TextField(max_length=128)
+    query = models.TextField()
+    query_sort = models.CharField(
+        max_length=16, default=SortOptions.DATE, choices=SortOptions.as_choices(), null=True
+    )
+    position = models.PositiveSmallIntegerField(null=False)
+    org_member_id = FlexibleForeignKey("sentry.OrganizationMember", null=True)
```

Comment:
Nit: Just want to mention this even though you're removing it - this should be `org_member` since you're using a fk here.